docs/dex-specification.md

```diff
@@ -192,25 +192,33 @@ The contract also manages protocol-wide charge recipient. The `feeTo` defines th
 
 <!-- github table -->
 
-|          Function           |                                Description                                |
-| --------------------------- | ------------------------------------------------------------------------- |
-| [`createPair`](#createpair) | Create a pair for two given tokens if such a pair doesn't already exist.  |
-| `getPair`                   | Return the address of the pair for two given tokens if such a pair exist. |
-| `setFeeToSetter`            | Set the new address of the protocol-wide charge recipient.                |
-| `setFeeTo`                  | Allow a provided address to control protocol-wide charge recipients.      |
-| `allPairsLength`            | Return the number of created pairs.                                       |
+|          Function           |                                                                        Description                                                                        |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`createPair`](#createpair) | Create a pair for two given tokens if such a pair doesn't already exist.                                                                                  |
+| `getPair`                   | Return the address of the pair for two given tokens if such a pair exist.                                                                                 |
+| `setFeeToSetter`            | Set the new address of the protocol-wide charge recipient. Note that a mistake in the address can result in the loss of control over the fee destination. |
```

Comment:
feeToSetter is not the recipient of a protocol-wide charge, it is a feeToo address. setFeeTooSetter sets the address which is allowed to **control** protocol-wide charge recipients and can only be called by the previous `feeToSetter`. Setting up a wrong `feeToSetter` address leads to the loss of an address that controls the `feeToo` address which is an address of the protocol-wide charge recipient